ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -783,27 +788,28 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift
-- as: aws-ipi-ovn-ipsec-p1-f4
-  cron: 46 7 2,6,10,14,18,22,27 * *
+- as: aws-ipi-ovn-ipsec-fips-p1-f4
```

Comment:
I think we can give up the transition to fips mode per the discussion in https://docs.google.com/spreadsheets/d/1mgPxhb4iUsPQfItdj9rzHGL0-iLY6GNFs5pijIW7G6o/edit?disco=AAAAyP2rHTE